x-pack/legacy/plugins/siem/public/components/charts/barchart.test.tsx

```diff
@@ -168,163 +268,29 @@ describe('BarChartBaseComponent', () => {
   });
 });
 
-describe.each([
-  [
-    [
-      { key: 'uniqueSourceIps', value: [{ y: 1714, x: 'uniqueSourceIps' }], color: '#DB1374' },
-      {
-        key: 'uniqueDestinationIps',
-        value: [{ y: 2354, x: 'uniqueDestinationIps' }],
-        color: '#490092',
-      },
-    ],
-  ],
-  [
-    [
-      { key: 'uniqueSourceIps', value: [{ y: 1714, x: '' }], color: '#DB1374' },
-      {
-        key: 'uniqueDestinationIps',
-        value: [{ y: 2354, x: '' }],
-        color: '#490092',
-      },
-    ],
-  ],
-  [
-    [
-      { key: 'uniqueSourceIps', value: [{ y: 1714, x: 'uniqueSourceIps' }], color: '#DB1374' },
-      {
-        key: 'uniqueDestinationIps',
-        value: [{ y: 0, x: 'uniqueDestinationIps' }],
-        color: '#490092',
-      },
-    ],
-  ],
-])('BarChartWithCustomPrompt', mockBarChartData => {
+describe.each(chartDataSets)('BarChart', data => {
   let shallowWrapper: ShallowWrapper;
+
   describe('renders barchart', () => {
     beforeAll(() => {
-      shallowWrapper = shallow(
-        <BarChartWithCustomPrompt
-          height={customHeight}
-          width={customWidth}
-          data={mockBarChartData}
-        />
-      );
+      shallowWrapper = shallow(<BarChart configs={mockConfig} barChart={data} />);
     });
 
-    it('render BarChartBaseComponent', () => {
-      expect(shallowWrapper.find(BarChartBaseComponent)).toHaveLength(1);
-      expect(shallowWrapper.find(ChartHolder)).toHaveLength(0);
+    it(`with data ${JSON.stringify(data)}`, () => {
```

Comment:
I'm hesitant to use a `JSON.stringify` just to get a log here. Too expensive for poor readability. maybe just log the index, and if the test fails the engineer can go look at which object it is?

Reply:
Good catch, that's definitely a concerned, I tried applying `%j` from https://jestjs.io/docs/en/api#describeeachtable-name-fn-timeout but didn't work. Will give it another try in the update, otherwise I'll just log it.